src/auth/admin.service.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@nestjs/common';
+
+import { Params } from '../models/params.model';
+import { User } from './user.entity';
+import { UserRepository } from './user.repository';
+
+@Injectable()
+export class AdminService {
+
+  public constructor(private readonly userRepository: UserRepository) {
```

Comment:
always write the constructor like this even if only one arg cause when you add a new one youll have to change the whole constructor while this way youll only have to append a new line
```
public constructor(
  private readonly userRepository: UserRepository,
) {
}
```

Reply:
deal